.storybook/main.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  stories: [
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
WDYT of colocating component stories? We could use more coherent organization for the shared stuff, and if we're doing that for the stories, we might as well for the components too

Reply:
I think it's a great idea when components follow a certain convention, but we have to come up with a minimum viable implementation that will allow devs to transition to it. Maybe we should do both: in `stories` AND in `components`, once we establish some conventions.